etna/whatson/models.py

```diff
@@ -1179,18 +1030,13 @@ class Meta:
     subpage_types = []
 
     @cached_property
-    def price_range(self):
+    def price_label(self):
```

Comment:
Not sure if we need this property. I can compute this in the frontend based on `price`.